### DISCUSSION TITLE
Why do you use a LazyColumn rather than a Column with verticalScroll modifier in logins template?

### DISCUSSION THREAD

**Discussion by danielesegato** [Gurupreet/ComposeCookBook#60] [Q&A]
Your login template uses a LazyColumn with 1 item for each ui component (labels, text fields etc.)

I was wondering if there's any particular reason I'm missing for that: sure you might go out of the screen with some component at any given time, but other than that the number of items is fixed and not likely to grow.

I would have probably just use a Column with a verticalScroll modifier, if you'd want to convince me to use a LazyColumn instead what would you say? :)

Thanks (nice repository)

**Comment by Gurupreet** [comment]
Hey @danielesegato Thanks for the great question. 
Yes I started with LazyColumn because I was planning to make following UI swipe up after success login considering onboarding would have top to down scroll flow. But I remember I was having some issue auto scrolling down to particular item in LazyColumn. So I did initial setup with LazyColumn as you won't want bottom UI to only render unless needed.

But YES, in Login page case we can use verticalScroll with just plain Column. Using LazyColumn is not necessary at all.
